src/App.scss

```
@@ -1 +1,12 @@
-// styles go here
+body {
+  font-family: system-ui, "Segoe UI", Roboto, sans-serif;
+  margin: 0 auto;
+  max-width: 60rem;
+  padding: 0.5rem;
+}
+
+.App {
+  background: #eee;
+  margin-bottom: 2rem;
+  padding: 0.5rem 1rem;
```

Remember about proper style order
```suggestion
  margin-bottom: 2rem;
  padding: 0.5rem 1rem;
  background: #eee;
```